student.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './person'
+
+class Student < person
```

Comment:
- Hey, please, let's kindly ensure that when setting the classroom for a student, it should also add it to the classrooms or the student like this below:

```ruby
class Student < Person
  attr_reader :classroom
```